.devcontainer/docker-compose.yaml

```diff
@@ -43,6 +43,30 @@ services:
     volumes:
       - "strapi-data:/var/lib/postgresql/data"
 
+  app:
```

Comment:
I think you have the changes from #38 here. Have you checked out from `37-add-nextjs-app-to-devcontainers` instead of the current `main`? Consider rebasing so that we have a distinct set of changes 👍

Reply:
Yes, this branch is based on the branch from #38

Reply:
I see. I think ensuring that you checkout from the latest main makes the PR more readable and independent (e.g. we can edit changes for a specific feature without requiring a later merge / rebase conflict resolution).